delatore/outputs/telegram/parsing.py

```diff
@@ -12,21 +13,23 @@ class CommandParsingError(Exception):
     """Error during command parsing"""
 
 
-def parse_command(message):
+@dataclass
+class ParsedStatusCommand:
+    target: str
+    detailed: str = ''
+    depth: int = 1
+
+    def __post_init__(self):
+        self.depth = int(self.depth)
+
+
+def parse_command(message) -> ParsedStatusCommand:
     """Parsing command arguments to arguments list"""
     LOGGER.debug('Got message: %s', message)
     try:
-        cmd, target, *args = shlex.split(message)
-    except ValueError:
-        raise CommandParsingError('No target provided for `/status` command')
-    depth = None
-    detailed = args.pop(0) if args else None
-    try:
-        depth = int(args.pop(0))
-    except IndexError:
-        depth = 1
+        _, target, *args = shlex.split(message)
+        return ParsedStatusCommand(target, *args)
     except ValueError:
-        raise CommandParsingError(f'Invalid depth value: {depth}')
-    if args:
-        raise CommandParsingError(f'Command {cmd} got unexpected arguments: {args}')
-    return target, detailed, depth
+        raise CommandParsingError('Incorrect `/status` command')
```

Comment:
When does this happen? Is this case covered in test?

Reply:
It happens when we try to send /status command with `_`